docs/wg/research/chromium/svg/animation-and-smil.md

```
@@ -0,0 +1,253 @@
+| ------------------------------- | ------------------------------------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------- |
+| SMIL                            | SVG attributes (CSS properties or non-CSS like `points`, `d`, `viewBox`) | `animVal` slot (see [animated-properties-idl.md](./animated-properties-idl.md)), then folded into `ComputedStyle` for presentation attributes |
+| CSS animations / Web Animations | CSS properties                                                           | `ComputedStyle` directly via the standard `core/animation/` engine                                                                            |
+
+When both engines target the same property, **SMIL's `animVal` wins** and the
+CSS animation effect is suppressed for that property. Implemented by sampling
+SMIL first and gating CSS animation effect application on whether SMIL is
+contributing for the attribute.
+
+## SMIL data model
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)